pages/sdk/data-hooks/useGames.mdx

```diff
@@ -0,0 +1,85 @@
+import { Callout } from 'components'
+
+Used for fetch pre-match games.
+
+Hook represents a logic wrapper over standard Apollo's `useQuery` hook. Explore [Apollo's docs](https://www.apollographql.com/docs/react/api/react/hooks#usequery) to understand what data the hooks return.
+
+You can find the list of available sports [here](https://thegraph.azuro.org/subgraphs/name/azuro-protocol/azuro-api-polygon-v2/graphql?query=%7B%0A++sports+%7B%0A++++name%0A++%7D%0A%7D).
+
+#### Usage
+
+```ts
+import { useGames, Game_OrderBy, OrderDirection } from '@azuro-org/data'
+
+const { loading, error, data } = useGames(props)
+
+const games = data?.games
+```
+
+#### Props
+
+```ts
+{
+  filter?: {
+    limit?: number // limit the number of rows returned from a query
+    offset?: number // omit a specified number of rows before the beginning of the result set
+    sportName?: string // returns games from specific sport
```

Comment:
wrong param, should be `sportSlug` (the same typo in sdk readme)

Reply:
5836f805b245b40d658f9236f6dcc69334290cb2